Homework/2/Program.cs

```diff
@@ -0,0 +1,390 @@
+﻿using System;
+
+namespace Homework2
+{
+    class Program
+    {
+        //Create a method for each operation:
+        //Sort an array
+        //Add element at the start of an array
+        //Add element at the end of an array
+        //Add element at any position of an array
+        //Remove element at the start of an array
+        //Remove element at the end of an array
+        //Remove element at a given index of an array
+        //Login: takes username and password.If they match a hidden username and password- a message "Logged in!" is printed.
+        //Username capitalization does not matter, password capitalization must match.
+
+        //Variables
+        private static bool loggedIn = false; // Variable to track if user is "logged in"
+        private static string[][] userArray = new string[5][]; // array of user arrays with login info which is string[username,password]; Would rather use List<>
+        
+
+        static void Main(string[] args)
+        {
+            //Adding a couple of users to array
+            userArray[0] = new string[] { "Admin", "slAptazodis" };
+            userArray[1] = new string[] { "user1", "pSwD" };
+            userArray[2] = new string[] { "LaIMONAS", "123" };
+            userArray[3] = new string[] { "Xavier", "Dust" };
+            userArray[4] = new string[] { "Blaber", "Lol" };
```

Comment:
Some cases don't add to anything extra and few of them would be enough to check if capitalization is truly being ignored.

Reply:
Be minimalistic.